app/form_builders/mps_form_builder.rb

```diff
@@ -1,4 +1,10 @@
 class MpsFormBuilder < GovukElementsFormBuilder::FormBuilder
+  def error_messages(options = {})
```

Comment:
📓  This helper allows us to use an inline display of the errors in the context of the form, as such:

```ruby
form_for @object do |f|
  = f.error_messages
end
```

and supports all the additional options that the GovukElementsErrorsHelper supports as well.